stories/useSet.story.tsx

```diff
@@ -13,6 +13,7 @@ const Demo = () => {
       <button onClick={() => remove('hello')} disabled={!has('hello')}>
         Remove 'hello'
       </button>
+      <button onClick={() => toggle('hello')}>{has('hello') ? 'Remove' : 'Add'} 'hello'</button>
```

Comment:
While it's a neat way to demonstrate the `has` method in code it wasn't immediately clear this button was the toggle button from looking at the UI. In my opinion the button label `Toggle 'hello'` is more clear. The `has` method could be demonstrated in a different way using an input field for example.

Reply:
I simplified the example so it's focused on the use of `toggle` 🙂